src/pages/donate.js

```diff
@@ -66,7 +66,9 @@ const Donate = ({ componentName }) => {
 								We love our lattes. We also love knowing that our youth are
 								prepared for the future. $5 matters. You matter. Let’s do this.
 							</p>
+							<a href = "https://paypal.me/treobytes" target = "_blank">
```

Comment:
don't add spaces between the attributes, just write them as 
href="https://paypal.me/treobytes" target="_blank"

you run the risk of getting weird side effects